README.md

```diff
@@ -18,6 +18,8 @@ npm install -g fleetctl
 fleetctl preview
 ```
 
+> Running into an error?  Make sure you're not using `sudo` when you do `npm install`.
```

Comment:
```suggestion
> Running into an error in which your machine does not recognize `fleetctl`?  Make sure you're not using `sudo` when you run `npm install fleetctl`.
```
Added more information about the error. A couple additional changes.

Reply:
@noahtalerman Turns out sometimes `sudo npm install` is the only option for folks, so my advice here is actually not generally applicable.  I'll just close this PR and we can tackle specific install errors folks run into on a case by case basis